frontend/src/views/ComplianceReports/buttonConfigs.jsx

```diff
@@ -43,12 +44,12 @@ export const containedButton = (label, startIcon) => ({
 export const buttonClusterConfigFn = ({
   hasRoles,
   currentUser,
-  methods,
   t,
   setModalData,
   updateComplianceReport,
   reportData,
-  isGovernmentUser
+  isGovernmentUser,
+  isSigningAuthorityDeclared
 }) => {
   const reportButtons = {
     saveDraft: {
```

Comment:
It's good to have some action or notification once a save to draft button is clicked.

Reply:
added a success on draft save alert